MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -109,9 +168,44 @@ public static void Process(GameSession session, string command)
             }
         }
 
+        private static void ProcessGotoCommand(GameSession session, string name)
+        {
+            Player target = GameServer.Storage.GetPlayerByName(name);
+            if (target == null)
+            {
+                session.SendNotice($"Couldn't find player with name: {name}!");
+                return;
+            }
+
+            IFieldObject<Player> fieldPlayer = target.Session.FieldPlayer;
+            if (target.MapId == session.Player.MapId && target.InstanceId == session.Player.InstanceId)
+            {
+                session.Send(UserMoveByPortalPacket.Move(session.FieldPlayer.ObjectId, fieldPlayer.Coord, fieldPlayer.Rotation));
+                return;
+            }
+            session.Player.Warp(mapId: target.MapId, coord: fieldPlayer.Coord, instanceId: target.InstanceId);
+        }
+
+        private static void ProcessOneshotCommand(GameSession session)
+        {
+            Player player = session.Player;
```

Comment:
Could get rid of this variable since it's not being used on line 199 when it could be